front/app/utils/moduleUtils.tsx

```diff
@@ -392,6 +392,8 @@ export type OutletsPropertyMap = {
     ) => (settingKey: string, settingValue: any) => void;
     errors: CLErrors;
   };
+  'app.containers.Admin.pages-menu.index': Record<string, any>;
+  'app.containers.Admin.pages-menu.NavigationSettings': Record<string, any>;
```

Comment:
But in other places, we also use `Record<string, any>`. Do you want me to change them too? Or it's fine to use different types in different places?

Reply:
You can if very limited in scope. Otherwise, it should be a different PR. In general, making code improvements is not an all-or-nothing. You can improve it in 1 place, communicate it to the team and make the other changes later/gradually.

Here it'll probably be rather small to adjust, so you could do it here or quickly do it in a separate PR.

Reply:
It seems I misunderstood your comment first. Indeed, it's better not to allow any props. Though `{}` gave me a lint error

```
  395:44  error  Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `Record<string, unknown>` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead  @typescript-eslint/ban-types
```

but `Record<string, never>` works fine. Thanks!

Reply:
Ah no, it doesn't work. Now it says
```
  Type '{ id: "app.containers.Admin.pages-menu.NavigationSettings"; }' is not assignable to type 'Record<string, never>'.
    Property 'id' is incompatible with index signature.
      Type 'string' is not assignable to type 'never'.
    13 |   return (
    14 |     <>
  > 15 |       <Outlet id="app.containers.Admin.pages-menu.NavigationSettings" />
       |        ^^^^^^
```

Reply:
I'll have a look tomorrow 😁

Reply:
You can leave it as is, it's too complicated to change in this PR.